docs/data/charts/pyramid-demo/pyramid.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Charts - Pyramid demonstration
+productId: x-charts
+components: FunnelChart, FunnelPlot
+---
+
+# Charts - Pyramid demonstration
+
+<p class="description">This page demonstrates the pyramid chart.</p>
+
+## Pyramid Chart
+
+The pyramid chart is a variation of the funnel chart.
+
+To create a pyramid chart, set the `curve` property to `pyramid` in the series.
+
+{{"demo": "Pyramid.js"}}
+
```

Comment:
We can also have a second example explaining data directionality, pyramid/upside-down-pyramid.